.github/workflows/release.yml

```diff
@@ -262,8 +271,12 @@ jobs:
       - name: Install dependencies (mac)
         if: runner.os == 'macOS'
         run: |
-          HOMEBREW_NO_AUTO_UPDATE=yes HOMEBREW_NO_INSTALL_CLEANUP=yes brew install sdl2 sdl2_image sdl2_ttf sdl2_mixer gettext ccache parallel
+          brew install gettext ccache parallel
+          ~/build-script/get-mac-sdl2-framework.sh
```

Comment:
typo: ~~build-script~~ **build-scripts**

Reply:
Yep, no such file or directory, this is it